README.md

```diff
@@ -118,12 +118,20 @@ Note on NVME storage: Samsung PM981 drives will not work out of the box if at al
 | SSDT-PTSWAK           | Fix sleep issues               |
 | SSDT-SBUS-MCHC        | SBUS fix                       |
 | SSDT-USBX             | USBX patch                     |
+| SSDT-BKey             | Fix display brightness keys    |
 
-## Pre-Install & BIOS settings
 
-First, read the [Dortania OC guide](https://dortania.github.io/OpenCore-Install-Guide/). The guide will take you through the creation of installation USB and drive formatting. Update to the latest firmware (the easiest way is to use fwupd on linux).
+## Pre-Install: Creating the installation USB stick
 
-Then do the following BIOS settings:
+First, read the [Dortania OC guide](https://dortania.github.io/OpenCore-Install-Guide/). The guide will take you through the creation of installation USB and drive formatting. Update the laptop to the latest BIOS firmware from your existing OS (the easiest way is to use fwupd on linux). Once using MacOS, it's a bit more complicated to update the BIOS.
```

Comment:
I wouldn't say it's more complicated to update the BIOS, you just have to boot to linux or windows to do it. This can of course break some things in macOS but that's hackintoshing for you.

Reply:
Yep... I've used a live cd to do it. Don't have multiple boot :)